tests/sqllogictests/suites/query/02_function/02_0065_function_json

```diff
@@ -63,6 +63,52 @@ SELECT json_array(true, 1, 'str', [1,2], {'k':'v'}, null)
 ----
 [true,1,"str",[1,2],{"k":"v"},null]
 
+query T
+SELECT json_object()
+----
+{}
+
+query T
+SELECT json_object('k1', 1, 'k2', 'str', 'k3', [1,2], 'k4', {'k':'v'}, 'k5', null)
```

Comment:
what`s the advance of this form?
why not  simply `select json_object({"k1":1,"k2":"str","k3":[1,2],"k4":{"k":"v"}})`